kong-ngx-build

```diff
@@ -213,10 +217,18 @@ main() {
       popd
     fi
 
-    if [[ $OPENRESTY_PATCHES == 1 && ! -d $OPENRESTY_PATCHES_DOWNLOAD ]]; then
-      warn "Kong OpenResty patches not found, downloading..."
+    if [[ $OPENRESTY_PATCHES != 0 ]]; then
       pushd $DOWNLOAD_CACHE
-        curl -s -S -L https://github.com/Kong/openresty-patches/archive/master.tar.gz | tar xz
+        if [[ ! -d $OPENRESTY_PATCHES_DOWNLOAD ]]; then
+          warn "Kong OpenResty patches not found, cloning..."
+          git clone https://github.com/Kong/openresty-patches $OPENRESTY_PATCHES_DOWNLOAD
+        fi
+
+        pushd $OPENRESTY_PATCHES_DOWNLOAD
+          notice "Checking out branch '$OPENRESTY_PATCHES' of Kong's OpenResty patches..."
+          git checkout $OPENRESTY_PATCHES
```

Comment:
Should probably use -b or -t along with -f or something here to always ensure we get the latest changes of a branch, or a newer branch, on as many git versions as possible.